app/src/main/java/com/infomaniak/swisstransfer/ui/screen/main/settings/SettingsDownloadsLimitScreen.kt

```diff
@@ -17,26 +17,13 @@
  */
 package com.infomaniak.swisstransfer.ui.screen.main.settings
```

Comment:
I updated the Settings too in another PR, so I'm looking at this one first, to see if it will cause issues with my PR.
After that, I'll accept this one then rebase mine afterward.